adafruit_mpu6050.py

```diff
@@ -347,3 +372,15 @@ def cycle_rate(self, value: int) -> None:
             raise ValueError("cycle_rate must be a Rate")
         self._cycle_rate = value
         sleep(0.01)
+
+    @property
+    def clock_source(self) -> int:
+        """Returns current CLKSEL value, Reg Map Pg 40"""
+        return self._clksel
+
+    @clock_source.setter
+    def clock_source(self, value: int) -> None:
+        """Sets CLKSEL value, Reg Map Pg 40"""
+        if (value < 0) or (value > 7):
```

Comment:
You could make this:

```suggestion
        if value not in range(8)
```

This will both use and iterator which is nice for memory, but will also implicitly check that the value is an integer!